### ISSUE TITLE
FoundryVTT v12 Support

### ISSUE THREAD

**Issue by Aeristoka** [jessev14/pf2e-staves#47]
This module is used almost religiously by several of my players, would love to see v12 Compatibility.

**Comment by Ghostdesu** [comment]
You might want to take a look at https://github.com/reonZ/pf2e-dailies which is fully maintained and up to date and has staves as part of its feature set

**Comment by Aeristoka** [comment]
> You might want to take a look at https://github.com/reonZ/pf2e-dailies which is fully maintained and up to date and has staves as part of its feature set

I'll have to look into that, does it allow for preparing more than one staff? And put a full spell list from the staff into the Player's spell list section?

**Comment by Ghostdesu** [comment]
>does it allow for preparing more than one staff? 

No, to my knowledge, it implements the RAW of 1 staff at a time.


>put a full spell list from the staff into the Player's spell list section?

Yes, it adds a new spell entry to the Activations tab with all the spells on the staff.
